src/envo/envo-edit.owl

```diff
@@ -39732,6 +39732,19 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000066> <http://purl.obolibrary.org/obo/ENVO_00000015>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01001497>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105101> (plastic)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://readanddigest.com/what-is-plastic/>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105101> "An environnmental material which 1) is composed primarily of polymers with high molecular mass and 2) once deformed does not return to its original shape."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105101> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105101> <https://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105101> <http://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105101> "2021-04-07T14:40:26Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(rdfs:comment "In German \"Kunststoff\" connotes those plastics which have intentionally been shaped by some art, thus \"Kunst\"."@en) <http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym> <http://purl.obolibrary.org/obo/ENVO_06105101> "Kunststoff"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym> <http://purl.obolibrary.org/obo/ENVO_06105101> "Plastica"@it)
```

Comment:
Hi @iwilkie,

Great to see some translations! For consistency within ENVO, please change has_related_synonym to has_exact_synonym for the translations.

Reply:
@raissameyer I think this is true in this case, but there will some translations that actually are broader / narrower / related in terms of the semantics behind the translated. Thus there are cases where these can be used.

Reply:
@pbuttigieg Agreed. In this case "Plastik" @de and "Plastics" @it would be `has_exact_synonyms`, while "Kunststoff"@de would be a `has_narrow_synonym` (as explained in @iwilkie's comment on this annotation)